src/block/blog-posts/index.php

```diff
@@ -206,13 +206,18 @@ function stackable_render_blog_posts_block( $attributes, $content ) {
 				$featured_image_src = $featured_image_urls[ $attributes['imageSize'] ];
 				if ( ! empty( $featured_image_src ) ) {
 					$featured_image = sprintf(
-						'<figure class="%s"><a href="%s"><img src="%s" alt="%s" width="%s" height="%s"/></a></figure>',
+						'<figure class="%s"><a href="%s">%s</a></figure>',
 						esc_attr( $featured_image_classes ),
 						esc_url( get_permalink( $post_id ) ),
-						esc_url( $featured_image_src[0] ),
-						esc_attr( get_the_title( $post_id ) ),
-						esc_attr( $featured_image_src[1] ),
-						esc_attr( $featured_image_src[2] )
+						get_the_post_thumbnail(
+							$post['ID'],
```

Comment:
What's the difference if we use this?
```suggestion
							$post_id,
```